docs/modules/customize/pages/styles.adoc

```diff
@@ -54,10 +54,9 @@ Notice that you can resize this textarea.
 
 Decidim uses Webpacker to compile assets. There are two ways to customize CSS:
 
-1. if you want to redefine just CSS vars or Foundation settings, use _decidim-settings.scss
```

Comment:
Shouldn't we explain how to redefine tailwind variables (i.e. through `tailwind.config.js`?)

Reply:
Moved to #11739